examples/DDIM/Project.toml

```diff
@@ -0,0 +1,19 @@
+[deps]
+Augmentor = "02898b10-1f73-11ea-317c-6393d7073e15"
+BSON = "fbb218c0-5317-5bc6-957e-2ee96dd4b1f0"
+BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
+CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
+Comonicon = "863f3e99-da2a-4334-8734-de3dacbe5542"
+Images = "916415d5-f1e6-5110-898d-aaa5f9f070e0"
+Lux = "b2108857-7c20-44ae-9111-449ecde12c47"
+MLUtils = "f1d291b0-491e-4a28-83b9-f70985020b54"
+NNlib = "872c559c-99b0-510c-b3b7-b6c96a88d5cd"
+Optimisers = "3bd65402-5787-11e9-1adc-39752487f4e2"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+ProgressBars = "49802e3a-d2f1-5c88-81d8-b72133a6f568"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
+Setfield = "efcf1570-3423-57d1-acb7-fd33fddbac46"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
```

Comment:
Last thing. Let's add compat bounds here and add `examples/DDIM` to https://github.com/avik-pal/Lux.jl/blob/7a3eebb5b4014bca80ca9baf064de013b2eaee8f/.github/workflows/CompatHelper.yml#L30 so that they are automatically updated in the future

Reply:
I added `examples/DDIM` in https://github.com/avik-pal/Lux.jl/pull/190/commits/e6c0f1532f714b2fa77ac95f939b1b5e3f73484f